classes/DataClass/Regmem/Category.php

```diff
@@ -9,14 +9,48 @@
 class Category extends BaseModel {
     public string $category_id;
     public string $category_name;
-    public ?string $category_description;
-    public ?string $legislation_or_rule_name;
-    public ?string $legislation_or_rule_url;
-    public EntryList $summaries;
-    public EntryList $entries;
+    public ?string $category_description = null;
+    public ?string $legislation_or_rule_name = null;
+    public ?string $legislation_or_rule_url = null;
+    public ?EntryList $summaries = null;
+    public ?EntryList $entries = null;
 
-    public function emoji() {
+    public function only_old_entries(string $register_date): bool {
+        foreach ($this->entries as $entry) {
+            if ($entry->isNew($register_date)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static function emojiLookup(string $value): string {
         $emoji_lookup = [
+            "Remuneration or other material benefit" => "💼",
+            "Membership/Chairmanship of bodies in receipt of Senedd funds" => "👥",
+            "Record of the employment of family members" => "👪",
+            "Directorships" => "👔",
+            "Gifts, hospitality, material benefit or advantage" => "🎁",
+            "Remunerated employment, office, profession etc" => "💼",
+            "Financial sponsorship" => "💳",
+            "Overseas visits" => "🌍",
+            "Land and property" => "🏠",
+            "Record of membership of societies" => "🤝",
+            "Donations and other support" => "💳",
+            "Gifts, benefits and hospitality" => "🎁",
+            "Visits" => "🌍",
+            "Shareholdings" => "📈",
+            "Land and Property" => "🏠",
+            "Miscellaneous" => "🏷️",
+            "Unremunerated interests" => "🤝",
+            "Family members who benefit from Office Cost Expenditure" => "👪",
+            "Voluntary" => "🤝",
+            "Controlled transactions" => "📊",
+            "Gifts" => "🎁",
+            "Heritable property" => "🏠",
+            "Interest in shares" => "📈",
+            "Overseas visits" => "🌍",
+            "Remuneration and related undertaking" => "💼",
```

Comment:
I understand that this is to help indicate which categories are the equivalent across chambers but I'm not entirely convinced by it. However, I am old so maybe not the target market :)